### DISCUSSION TITLE
Daily 23-03-2022

### DISCUSSION THREAD

**Discussion by YaakovHatam** [YaakovHatam/diet-recipes#15] [Show and tell]
Daily

**Comment by ShlomoRussell** [comment]
I haven't gotten started yet. It seems to me that my part needs to be done after a few other ones have been completed. I plan on doing te bulk of it tomorrow be'h.

**Comment by Amicht** [comment]
I've finished my part, currently existing routes: Home,About, MyCart , and  pushed it to repo, just waiting to add more by team requests.

also, I added a 'Components' folder with routs-file names, just to prevent errors and Allow react to run successfully.
I will be available tommorow for more assingnmets and help.

**Comment by OsnatDayan** [comment]
אני נמצאת ממש יחסית בסיום המשימה נשארו לי כמה דברים שאני יברר מול יעקב ובעז"ה מקווה לסיים מחר

**Reply by ShlomoRussell** [reply]
@qpasdmx אפשר בבקשה להעלות את מה שעשית כבר

**Reply by OsnatDayan** [reply]
אם תוכל להגיד לי מה המייל שלך אני ישלח לך למייל אני פשוט רוצה להעלות את זה כשזה יהיה גמור

**Reply by ShlomoRussell** [reply]
Baapteshuva@gmail.com
